blobfuse/blobfuse.cpp

```diff
@@ -304,6 +306,18 @@ int read_config(const std::string configFile)
     }
 }
 
+void destroyBlobfuseOnAuthError()
+{
+    char errStr[] = "Unmounting blobfuse.\n";
+    syslog(LOG_ERR, errStr, sizeof(errStr));
+    ssize_t n = write(stdErrFD, errStr, sizeof(errStr));
+    if (n == -1) {
+        syslog(LOG_ERR, "Failed to report back failure.");
+    }
+    close(stdErrFD);
+    fuse_unmount(config_options.mntPath.c_str(), NULL);
+}
+
 int configure_tls();
 void *azs_init(struct fuse_conn_info * conn)
 {
```

Comment:
kernel_version < 4.16 can be replaced by blobfuse_constants.min_kernel_version or equivalent

Reply:
Will change accordingly.